pkg/api/frontend_logging.go

```diff
@@ -28,14 +66,121 @@ type frontendSentryEvent struct {
 	Exception *frontendSentryException `json:"exception,omitempty"`
 }
 
+func guessSourceMapLocation(sourceUrl string) (*sourceMapLocation, error) {
+	u, err := url.Parse(sourceUrl)
+	if err != nil {
+		return nil, err
+	}
+	if strings.HasPrefix(u.Path, "/public/build/") {
+		return &sourceMapLocation{
+			dir:      setting.StaticRootPath,
+			path:     path.Join("build", u.Path[len("/public/build/"):]) + ".map",
+			pluginId: "",
+		}, nil
+	} else if strings.HasPrefix(u.Path, "/public/plugins/") {
+		for _, route := range plugins.StaticRoutes {
+			pluginPrefix := path.Join("/public/plugins/", route.PluginId)
+			if strings.HasPrefix(u.Path, pluginPrefix) {
+				return &sourceMapLocation{
+					dir:      route.Directory,
+					path:     u.Path[len(pluginPrefix):] + ".map",
+					pluginId: route.PluginId,
+				}, nil
+			}
+		}
+	}
+	return nil, nil
+}
+
+func getSourceMap(sourceUrl string) (*sourceMap, error) {
```

Comment:
In Go, initialisms should be written in consistent case.
```suggestion
func getSourceMap(sourceURL string) (*sourceMap, error) {
```

Reply:
I'll start remembering this, someday... :) 
- [ ] initialisims in all caps